core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -621,33 +621,34 @@ oppia.controller('StateResponses', [
                   stateCustomizationArgsService.savedMemento));
 
               AnswerClassificationService.getMatchingClassificationResult(
-                _explorationId, _state, answer, true,
-                rulesService).then(
-                    function(classificationResult) {
-                  var feedback = 'Nothing';
-                  var dest = classificationResult.outcome.dest;
-                  if (classificationResult.outcome.feedback.length > 0) {
-                    feedback = classificationResult.outcome.feedback[0];
-                  }
-                  if (dest === _stateName) {
-                    dest = '<em>(try again)</em>';
-                  }
-                  $scope.trainingDataAnswer = answer;
-                  $scope.trainingDataFeedback = feedback;
-                  $scope.trainingDataOutcomeDest = dest;
-
-                  var answerGroupIndex = classificationResult.answerGroupIndex;
-                  var ruleIndex = classificationResult.ruleIndex;
-                  if (answerGroupIndex !==
-                    _state.interaction.answerGroups.length &&
-                      _state.interaction.answerGroups[answerGroupIndex]
-                      .rules[ruleIndex].type !== RULE_TYPE_CLASSIFIER) {
-                    $scope.classification.answerGroupIndex = -1;
-                  } else {
-                    $scope.classification.answerGroupIndex = (
-                      classificationResult.answerGroupIndex);
+                _explorationId, _state, answer, true, rulesService)
+                .then(
+                  function(classificationResult) {
```

Comment:
I think you can save a level of indentation by moving this to the previous line?